x/staking/spec/01_state.md

```diff
@@ -59,9 +60,9 @@ When Tendermint reports evidence, it provides the validator address, so this
 map is needed to find the operator. Note that the `ConsAddr` corresponds to the
 address which can be derived from the validator's `ConsPubKey`.
 
-`ValidatorsByPower` is an additional index that provides a sorted list o
+`ValidatorsByPower` is an additional index that provides a sorted list of
 potential validators to quickly determine the current active set. Here
-ConsensusPower is validator.Tokens/10^6. Note that all validators where
+ConsensusPower is validator.Tokens/10^18. Note that all validators where
```

Comment:
This isn't true. Consensus power by default is reduced by `10^6` via the `PowerReduction` variable, which can be configured by an application.

I would rephrase this in such a manner. i.e. reduced by a default value of 10^6, which can be configurable.

Reply:
Hmm, you're right, don't know why I got mixed up.

```
// PowerReduction is the amount of staking tokens required for 1 unit of consensus-engine power
var PowerReduction = NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(6), nil))
```

Reply:
Is this configurable though? Looks hardcoded.

Reply:
I've updated the text.

Reply:
It's configurable -- it's a `var` which you can overwrite 👍